teos/src/main.rs

```diff
@@ -77,14 +82,19 @@ async fn main() {
     });
 
     // Load conf (from file or defaults) and patch it with the command line parameters received (if any)
-    let mut conf = config::from_file::<Config>(path.join("teos.toml"));
+    let mut conf = config::from_file::<Config>(conf_file_path);
     let is_default = conf.is_default();
     conf.patch_with_options(opt);
     conf.verify().unwrap_or_else(|e| {
         eprintln!("{e}");
         std::process::exit(1);
     });
 
+    // Log config file path and any non-default options
+    log::info!("Using configuration file: {:?}", &cloned_conf_file_path);
+    if !is_default {
+        log::info!("Using this non-default options.");
```

Comment:
As per issue #202 - we need to log the non-default options that are being used. However, the above line only specifies that the user is using non-default options without specifying what non-default options are being used.

Reply:
Indeed. This is not following the issue requirements.

You need to log a line for each option that is not being run as default.